src/cloud_radar/cf/unit/_template.py

```diff
@@ -250,9 +280,74 @@ def resolve_values(
                 )
                 for item in data
             ]
+        elif isinstance(data, str):
+            return self.resolve_dynamic_references(data)
         else:
             return data
 
+    def resolve_dynamic_references(self, data: str) -> str:
+        """
+        Replaces any dynamic references in the provided data with values from
+        our configuration.
+
+        Args:
+            data (str): the value to replace dynamic references within
+
+        Raises:
+            ValueError: If a dynamic reference has been used and no dynamic
+            references have been configured.
+            KeyError: If a dynamic reference name has been used and is not
+            found in the configuration
+        """
+
+        if "{{resolve:" in data:
+            print("For str may need to resolve SSM parameters: ", data)
```

Comment:
Are these debug prints? I'm not sure what all the library prints out. I think up til now I have not had much logging. Which I'm not sure is good either.

Reply:
Yes, good catch. I was having a few issues tracking code paths & regex. 

I'll take them out later on when I'm free

Reply:
That's a change pushed that removes the stray print statements & fixes the last of the flake8 issues